src/xwayland/XWM.cpp

```diff
@@ -890,6 +890,9 @@ CXWM::~CXWM() {
 
     if (eventSource)
         wl_event_source_remove(eventSource);
+
+    for (auto const& sr : surfaces)
```

Comment:
{}

Reply:
added, didn't realize it was necessary for loops mb